### PR TITLE
Add netstandard2.0 target to squirrel

### LINKED ISSUE #1649: Squirrel namespace not available in .NET Core
**Squirrel version(s)**
2.0.0 and 1.9.1

**Description**
The Squirrel namespace is not available, after squirrel.windows was installed.

**Steps to recreate**
```
mkdir testapp
cd testapp
dotnet new wpf
dotnet add package squirrel.windows --version 2.0.0
dotnet build
```
Open the project with VS2019 and try to access the namespace.

**Actual behavior**
Cannot access Squirrels namespace.

### PR DESCRIPTION
**Pull request by AArnott**
Fixes #1649

### CODE REVIEW

**Comment by robmen** [comment]
Is this PR dependent on another PR that removes the dependency on `DeltaCompressionDotNet.MsDelta.dll`?

**Comment by AArnott** [comment]
@robmen: no. That dll apparently merged into the main one with the 2.0.0 version of the dependency. It simply doesn't exist any more.

BTW, until https://github.com/taspeotis/DeltaCompressionDotNet/issues/17 is fixed, I have a follow up change after this PR to fully eliminate package restore warnings for netstandard/netcore apps that consume squirrel:
https://github.com/AArnott/Squirrel.Windows/compare/addNetStandardTarget...AArnott:dropDelta

**Comment by robmen** [comment]
Okay. I was just trying to (cursorily) understand why this PR build broke. Looks like it's fixed now.

**Review by robmen — APPROVED** [review]

File: src/Squirrel/Squirrel.csproj [hunk 1]
```diff
@@ -2,7 +2,7 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>net45</TargetFramework>
+    <TargetFrameworks>net45;netstandard2.0</TargetFrameworks>
```
Comment (GrahamTheCoder) [inline src/Squirrel/Squirrel.csproj:5]: From the [build log](https://dev.azure.com/squirrel-installers/Squirrel.Windows/_build/results?buildId=152&view=logs&j=12f1170f-54f2-53f3-20dd-22fc7dff55f9&t=944b7e9c-00d2-5ab3-4f94-8de8fe321ef4), it looks like this does get built, but it doesn't seem to get published as an artifact

Comment (GrahamTheCoder) [reply src/Squirrel/Squirrel.csproj:5]: Yeah the lack of nupkg surprised me too. I'll see if I can find time to PR something in that direction, though don't let that stop anyone else!

Comment (anaisbetts) [reply src/Squirrel/Squirrel.csproj:5]: It's me! It's my dev box! And it runs on literally millions of machines around the world and always has! 😅

**Pull request merged**